packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -110,14 +169,14 @@ class BottomSheet extends Component {
 			);
 		}
 
-		this.keyboardWillShowListener = Keyboard.addListener(
-			'keyboardWillShow',
-			this.keyboardWillShow
+		this.keyboardShowListener = Keyboard.addListener(
+			Platform.OS === 'ios' ? 'keyboardWillShow' : 'keyboardDidShow',
+			this.keyboardShow
 		);
 
-		this.keyboardDidHideListener = Keyboard.addListener(
-			'keyboardDidHide',
-			this.keyboardDidHide
+		this.keyboardHideListener = Keyboard.addListener(
+			Platform.OS === 'ios' ? 'keyboardWillHide' : 'keyboardDidHide',
+			this.keyboardHide
```

Comment:
`Will` keyboard events are not available on Android ([documentation reference](https://reactnative.dev/docs/0.61/keyboard#addlistener)).

Reply:
It may be worthwhile leaving a code comment inline stating this.

Reply:
Good idea! I've added the comment in this [commit](https://github.com/WordPress/gutenberg/pull/30291/commits/467e1b2926064ce452373fa510c8b778111cbb1a).